pkg/certificates/authorizations/authorizations.go

```diff
@@ -0,0 +1,183 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package authorizations
```

Comment:
moved from `certificates` package

Reply:
Nice, I hadn't seen that yet. :+1:

Reply:
What about `certificates`? `certificate` just seems weird imo. Mabye we need a better package name in general.

Reply:
but then it will be `pkg/certificates/certificates`?

Reply:
Sorry, I wasn't clear. I was asking about the existing `certificates` package, it is also a plural.

Reply:
ah sorry, I thought that you were talking about this package.

I use the singular, "seems weird" isn't a good reason for making it plural.

I've found sometimes some packages to sound better in the plural, however, I've chosen to be stuck on the convention.

Reasons, why a package name is in the plural, is for example that it clashes with a built-in type name, for example [`bytes`](https://golang.org/pkg/bytes/) and [`errors`](https://golang.org/pkg/errors/).